instance_pool.go

```diff
@@ -83,82 +83,80 @@ type UpdateInstancePool struct {
+// Response returns an empty structure to unmarshal an Instance Pool scaling API response into.
 func (ScaleInstancePool) Response() interface{} {
-	return new(ScaleInstancePoolResponse)
+	return new(BooleanResponse)
 }
 
-// DestroyInstancePool represents an instance pool destruction API request.
+// DestroyInstancePool represents an Instance Pool destruction API request.
 type DestroyInstancePool struct {
 	ID     *UUID `json:"id"`
 	ZoneID *UUID `json:"zoneid"`
 	_      bool  `name:"destroyInstancePool" description:"Destroy an Instance Pool"`
 }
 
-// DestroyInstancePoolResponse represents an instance pool destruction API response.
-type DestroyInstancePoolResponse struct {
-	Success bool `json:"success"`
-}
-
-// Response returns an empty structure to unmarshal an instance pool destruction API response into.
+// Response returns an empty structure to unmarshal an Instance Pool destruction API response into.
 func (DestroyInstancePool) Response() interface{} {
-	return new(DestroyInstancePoolResponse)
+	return new(BooleanResponse)
 }
 
-// GetInstancePool retrieves an instance pool's details.
+// GetInstancePool retrieves an Instance Pool's details.
 type GetInstancePool struct {
 	ID     *UUID `json:"id"`
 	ZoneID *UUID `json:"zoneid"`
 	_      bool  `name:"getInstancePool" description:"Get an Instance Pool"`
 }
 
-// GetInstancePoolResponse get instance pool API response.
+// GetInstancePoolResponse get Instance Pool API response.
 type GetInstancePoolResponse struct {
 	Count         int
 	InstancePools []InstancePool `json:"instancepool"`
 }
 
-// Response returns an empty structure to unmarshal an instance pool get API response into.
+// Response returns an empty structure to unmarshal an Instance Pool get API response into.
 func (GetInstancePool) Response() interface{} {
 	return new(GetInstancePoolResponse)
 }
 
-// ListInstancePools represents a list instance pool API request.
+// ListInstancePools represents a list Instance Pool API request.
 type ListInstancePools struct {
 	ZoneID *UUID `json:"zoneid"`
 	_      bool  `name:"listInstancePools" description:"List Instance Pools"`
 }
 
-// ListInstancePoolsResponse represents a list instance pool API response.
+// ListInstancePoolsResponse represents a list Instance Pool API response.
 type ListInstancePoolsResponse struct {
 	Count         int
 	InstancePools []InstancePool `json:"instancepool"`
 }
 
-// Response returns an empty structure to unmarshal an instance pool list API response into.
+// Response returns an empty structure to unmarshal an Instance Pool list API response into.
 func (ListInstancePools) Response() interface{} {
 	return new(ListInstancePoolsResponse)
 }
+
+// EvictInstancePoolMembers represents an Instance Pool members eviction API request.
+type EvictInstancePoolMembers struct {
+	ID        *UUID  `json:"id"`
+	ZoneID    *UUID  `json:"zoneid"`
+	MemberIDs []UUID `json:"memberids"`
```

Comment:
Just to be consistent across all fields of the struct:

```suggestion
	MemberIDs []*UUID `json:"memberids"`
```

Reply:
In this case a slice of pointer is not needed ;) let's stay consistent with the rest of the structure request, like
`UpdateVMAffinityGroup`...etc and many others.